ramls/check-in-by-barcode-response.json

```diff
@@ -115,6 +115,10 @@
     "staffSlipContext": {
       "description": "Context for staff slip tokens",
       "type": "object"
+    },
+    "inHouseUse": {
+      "type": "boolean",
+      "description": "Defines whether the item is in-house use or not."
```

Comment:
I don't think it is the item that is in-house use, it is whether the check-in is considered in-house use. Please could this be confirmed with Emma and reworded if needed.